wiki/osu!api/id.md

```diff
@@ -0,0 +1,11 @@
+# osu!api
+
+osu! memiliki API publik yang dapat digunakan oleh layanan pihak ketiga untuk memperluas dan mendukung platform osu!. Dulu API bersifat lebih tertutup, hanya dapat digunakan sesuai permintaan, tetapi sekarang siapa pun dapat menggunakan API selama mereka memiliki akun osu!.
+
+Anda dapat menemukan dokumentasi API di [Wiki osu!api](https://github.com/ppy/osu-api/wiki). Versi API saat ini menggunakan kunci pribadi untuk mengautentikasi permintaan. Anda dapat mengajukan permohonan kunci API di [halaman aplikasi](https://osu.ppy.sh/p/api) pada situs web osu!.
```

Comment:
kunci api = API Token right ? use `API Token` instead :/ 
avoiding misunderstanding for third party app developers

Reply:
I translated API Token before on an article that i can't remember, will `Token API` work?